code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -316,50 +364,49 @@
 /datum/reagent/consumable/ethanol/beepsky_smash
 	name = "Beepsky Smash"
 	id = "beepskysmash"
-	description = "Deny drinking this and prepare for THE LAW."
+	description = "Drink this and prepare for the LAW."
 	color = "#664300" // rgb: 102, 67, 0
-	boozepwr = 25
+	boozepwr = 90 //THE FIST OF THE LAW IS STRONG AND HARD
 	metabolization_rate = 0.8
 
 /datum/reagent/consumable/ethanol/beepsky_smash/on_mob_life(mob/living/M)
 	M.Stun(2)
-	..()
-	return 1
+	return ..()
```

Comment:
no, the previous code was fine, since Stun is always called we must always return 1, it doesn't matter what ..() returns.

Also, the M.Stun(2) above should be `M.Stun(2, 0)`, I must've missed it when I did the change.